submissions/2024-12-12-OpenFF-Sage-2.0.0-Training-Optimization-Dataset-v1.0/generate-combined-dataset.py

```diff
@@ -1,18 +1,19 @@
 
+import numpy as np
+import requests
+import logging
+logging.getLogger("openff").setLevel(logging.ERROR)
+
 from openff.qcsubmit.results import OptimizationResultCollection
 from openff.qcsubmit.datasets import OptimizationDataset
 from openff.qcsubmit.factories import OptimizationDatasetFactory
 from openff.qcsubmit.common_structures import MoleculeAttributes
-import numpy as np
+
 from openff.toolkit.utils import OpenEyeToolkitWrapper, ToolkitRegistry
 from openff.units import unit
-import logging
-logging.getLogger("openff").setLevel(logging.ERROR)
 
-filtered_and_combined = OptimizationResultCollection.parse_file(
-    "/Users/jennifer.clark/bin/openff-sage/data-set-curation/quantum-chemical/data-sets/1-2-0-opt-set-v3.json",
-#    "/Users/jennifer.clark/bin/openff-sage/data-set-curation/quantum-chemical/data-sets/1-2-0-td-set.json",
-)
+file = requests.get("https://raw.githubusercontent.com/openforcefield/openff-sage/refs/heads/main/data-set-curation/quantum-chemical/data-sets/1-2-0-opt-set-v3.json")
```

Comment:
If you want to super, super pedantic, you can use the GitHub permalink: https://raw.githubusercontent.com/openforcefield/openff-sage/37a36e7eeaf6cdca795847089a288bdff168c08a/data-set-curation/quantum-chemical/data-sets/1-2-0-opt-set-v3.json. It's kinda annoying to get this (click the `...` in the upper right, copy permalink, then click `raw` for the raw file at the permalink), but it should always point to this exact file version instead of whatever `main` happens to be. This is especially unlikely to matter here because this repo is basically an archive anyway, though.